onnx/backend/test/stat_coverage.py

```diff
@@ -116,7 +124,80 @@ def gen_node_test_coverage(schemas, f, ml):
 def gen_model_test_coverage(schemas, f, ml):
     # type: (Sequence[defs.OpSchema], IO[Any], bool) -> None
     f.write('# Model Test Coverage\n')
-    f.write('## To be filled.\n')
+    # Process schemas
+    schema_dict = dict()
+    for schema in schemas:
+        schema_dict[schema.name] = schema
+    # Load models from each model test using Runner._prepare_model_data
+    # TODO: Abstract due to similarity to Runner
+    # Need to grab associated nodes
+    attrs = dict()
+    for rt in load_model_tests(kind='real'):
+        model_dir = Runner._prepare_model_data(rt)
+        model_pb_path = os.path.join(model_dir, 'model.onnx')
+        model = load(model_pb_path)
+        f.write('## {}\n'.format(model.graph.name))
+        # Deconstruct model
+        num_covered = 0
+        for node in model.graph.node:
+            if node.op_type in common_covered or node.op_type in experimental_covered:
+                num_covered += 1
+                # TODO: Add details of which nodes are/aren't covered
+                # Iterate through and store each node's attributes
+                # TODO: Verify enum is being properly referenced here
+                for attr in node.attribute:
+                    if node.op_type not in attrs:
+                        attrs[node.op_type] = dict()
+                    if attr.name not in attrs[node.op_type]:
+                        attrs[node.op_type][attr.name] = []
+                    if attr.type == 1:
```

Comment:
Use the ENUM in proto please,

Reply:
Was quite confused as to how to do this; will discuss soon